src/openrct2/ReplayManager.cpp

```diff
@@ -537,6 +538,27 @@ namespace OpenRCT2
                     result.action->SetFlags(command.ebx & 0xFF);
                     break;
                 }
+                case GAME_COMMAND_MODIFY_TILE:
+                {
+                    int32_t param1 = command.edx;
+                    int32_t param2 = command.edi;
+                    CoordsXY loc = { (command.ecx & 0xFF) * 32, ((command.ecx >> 8) & 0xFF) * 32 };
```

Comment:
Untested, but:
```suggestion
                    auto loc = CoordsXY(command.ecx & 0xFF) * 32, ((command.ecx >> 8) & 0xFF) * 32);
```
is the cleanest way to get rid of the compile errors on Travis, I think.